### PR TITLE
Uplfe1031b

### PR DESCRIPTION
**Pull request by charlielizzy**
there was no ability to adjust font size or weight in the inputtext atom so i've added those props in so that for future components we can amend the size and weight as per design needs. the default remains the same so should have no impact on other uses on inputtext, but the option to adjust font size and weight from the default will now be there

### CODE REVIEW

File: src/components/atoms/InputText/InputText.tsx [hunk 1]
```diff
@@ -88,19 +97,21 @@ const InputText = forwardRef<HTMLInputElement, InputProps>(({ startIcon, endIcon
...
-      <Input startIcon={startIcon} endIcon={endIcon} {...rest} ref={ref} theme={theme} />
-      {endIcon && (
-        <IconWrapper startIcon={false} theme={theme}>
-          {endIcon}
-        </IconWrapper>
-      )}
-    </InputWrapper>
+    <>
+      <InputWrapper className={className}>
+        {startIcon && (
```
Comment (admmasters) [inline src/components/atoms/InputText/InputText.tsx:102]: 👀 we should use a ternary here instead

File: src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap [hunk 2]
```diff
@@ -22,7 +22,6 @@ exports[`<InputText /> renders the component disabled 1`] = `
   -webkit-transition: 0.2s ease-in-out;
   transition: 0.2s ease-in-out;
   font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
-  font-size: 16px;
```
Comment (admmasters) [inline src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap:25]: We should default to `font-size: 16px` so we don't break any consumers of the package

Comment (charlielizzy) [reply src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap:25]: this bit was automatically deleted when i ran -u to udpate the tests - do you want me to change it back?

File: src/components/types.ts [hunk 3]
```diff
@@ -31,6 +31,8 @@ export interface InputProps extends InputStatus, InputHTMLAttributes<HTMLInputEl
    * Slot for an icon on the right
    */
   endIcon?: ReactNode;
+
+  weight?: number | undefined;
```
Comment (moshie) [inline src/components/types.ts:35]: Don't think the undefined is needed here as `?:` indicates this is an optional param

**Review by moshie — CHANGES_REQUESTED** [review]

File: src/components/atoms/InputText/InputText.tsx [hunk 4]
```diff
@@ -46,15 +56,14 @@ const Input = styled.input<InputProps & { theme: AppTheme }>`
   padding: 0 16px;
   padding-left: ${({ startIcon }) => !!startIcon && '60px'};
   padding-right: ${({ endIcon }) => !!endIcon && '60px'};
-  font-size: ${typography.sizes.text.body};
-  font-weight: ${typography.weights.regular};
+  font-size: ${({ theme, size = 'body' }) => theme.typography.text.sizes[size]};
```
Comment (khumbon) [inline src/components/atoms/InputText/InputText.tsx:59]: Add a default font-size of 16px

File: src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap (hunk 2 above)
Comment (charlielizzy) [reply src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap:25]: in InputText the default is set to 'body' which is equal to 16px, so no change to the default in reality

File: src/components/atoms/InputText/InputText.tsx (hunk 4 above)
Comment (charlielizzy) [reply src/components/atoms/InputText/InputText.tsx:59]: 'body' is equal to 16px so there is still a default of 16px this way

**Review by admmasters — APPROVED** [review]
LGTM 🚀

**Review by moshie — APPROVED** [review]

**Pull request merged**